mapproxy/util/ext/odict.py

```diff
@@ -107,8 +107,8 @@ def __setitem__(self, key, value):
     :license: modified BSD license.
 """
 from __future__ import absolute_import
-from mapproxy.compat import iteritems
-from mapproxy.compat.itertools import izip, imap
+
+from past.builtins import cmp
```

Comment:
unused import?

Reply:
is it? cmp was a builtin in the past and now we need to import it if we want to use it. I did not look into how to replace it as it was only this one place.

Reply:
usage is here: https://github.com/mapproxy/mapproxy/pull/861/files/4337202090ec36b353dbb790b325b70b1541698a#diff-ddc40d8d20c2770cab74d4d3847fc1a8f178b463da616e6e13eac8041b0456f3L237